reference/api/tasks.md

```diff
@@ -261,4 +272,65 @@ Get a single task.
 
 ## Cancel tasks
 
+<RouteHighlighter method="POST" route="/tasks/cancel?{task_uid}"/>
+
+Cancel an `enqueued` or `processing` task based on `uid`, `status`, `type`, `indexUid`, or date. Task cancelation is an atomic transaction, **either all tasks are successfully canceled or none are**.
```

Comment:
I recommend a longer pause here because it gives a stronger sense of explanation/conclusion.

```suggestion
Cancel an `enqueued` or `processing` task based on `uid`, `status`, `type`, `indexUid`, or date. Task cancelation is an atomic transaction: **either all tasks are successfully canceled or none are**.
```